spec/spec_helper.rb

```diff
@@ -16,19 +16,19 @@
 
   config.before(:all) do
     ActiveRecord::Base.connection.tables.each do |table|
-      ActiveRecord::Base.connection.execute("DROP TABLE #{table}")
+      ActiveRecord::Base.connection.execute("DROP TABLE #{table} CASCADE")
```

Comment:
These changes are necessary to properly clean up the database after the new tests I've introduced (the foreign key related ones create dependent objects that block dropping the tables).